src/main/java/org/example/FileReadReplaceLettersWithCapitals.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class FileReadReplaceLettersWithCapitals {
+    public static String readTextFromFile(String filePath) throws IOException, IOException {
```

Comment:
a jeśli filePath będzie null? albo wartość w tekście będzie null? powinnaś wtedy ładnie to obsłużyć. Widzę, że używasz klauzuli throws ale dwa razy masz zadeklarowane to samo. Proponuje, żebyś z tymi zadaniami poczekała do poniedziałkowej lekcji. Ja już ci z rozpędu zaliczyłem kilka zadań z tej lekcji, ale nie uwzględniłem aspektu rzucania wyjątków. Poczekaj chwile z poprawą tego zadania jak powiem jak takie rzeczy obsługiwać w poniedziałek na lekcji ;)